### DISCUSSION TITLE
Тыква в честь Хэллоуина

### DISCUSSION THREAD

**Discussion by Ygoslavya** [Sokol-Games-GmbH/PixelCombats.PublicApi#381] [Ideas]
![Picsart_24-10-27_08-04-52-854](https://github.com/user-attachments/assets/c6a1db06-f516-4416-a458-d91969bee2e9)
![Picsart_24-10-27_08-05-30-626](https://github.com/user-attachments/assets/71cff9e7-11b2-4c08-a68d-f83153a1636d)
![Picsart_24-10-27_08-06-19-560](https://github.com/user-attachments/assets/ecd82267-decf-497e-bb38-07cdb72f7c9c)

**Comment by gat3v** [comment]
поздняк
